x-pack/packages/security/plugin_types_server/src/authentication/api_keys/api_keys.ts

```diff
@@ -185,6 +185,9 @@ export const crossClusterApiKeySchema = restApiKeySchema.extends({
         schema.arrayOf(
           schema.object({
             names: schema.arrayOf(schema.string()),
+            query: schema.maybe(schema.any()),
+            field_security: schema.maybe(schema.any()),
```

Comment:
We're relying on ES to validate the query and field_security syntax here.

Reply:
This is a complete list, and replication only contains names.
See https://github.com/elastic/kibana/issues/183682#issuecomment-2117829888